oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -73,6 +73,7 @@ public void testOperatingSystem() {
         assertTrue(proc.getStartTime() >= 0);
         assertTrue(proc.getBytesRead() >= 0);
         assertTrue(proc.getBytesWritten() >= 0);
+        assertTrue(proc.getOpenFiles()>= -1);

```

Comment:
my code formatter will add a space here.  Wouldn't hold up the commit for this but if you're making other changes, may as well prevent the space being added in the next unrelated commit :)